cdap-watchdog/src/test/java/co/cask/cdap/logging/appender/file/TestFileLogging.java

```diff
@@ -85,16 +85,48 @@ public static void setUpContext() throws Exception {
     txManager = injector.getInstance(TransactionManager.class);
     txManager.startAndWait();
 
-    LoggingContextAccessor.setLoggingContext(new FlowletLoggingContext("TFL_ACCT_1", "APP_1", "FLOW_1", "FLOWLET_1"));
-
     LogAppender appender = injector.getInstance(LogAppender.class);
     new LogAppenderInitializer(appender).initialize("TestFileLogging");
 
     Logger logger = LoggerFactory.getLogger("TestFileLogging");
-    for (int i = 0; i < 60; ++i) {
-      Exception e1 = new Exception("Test Exception1");
-      Exception e2 = new Exception("Test Exception2", e1);
-      logger.warn("Test log message " + i + " {} {}", "arg1", "arg2", e2);
+    Exception e1 = new Exception("Test Exception2");
+
+    LoggingContextAccessor.setLoggingContext(new FlowletLoggingContext("TFL_ACCT_2", "APP_1", "FLOW_1", "FLOWLET_1",
```

Comment:
nit:  Should we change these constants to use "NS" instead of "ACCT"? Only for the new ones.